code_challenges.rb

```diff
@@ -6,43 +6,73 @@
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
+def odds_sorted (array)
+    array_odds = []
+    i = array.length
+    while i > 0 do
+        i = i - 1
+        if array[i].is_a? Numeric and array[i].odd? 
+            array_odds << array[i]
+        end
+    end
+    array_odds.sort
+end
 
+# p odds_sorted(fullArr1)
 
 
 
 
 # --------------------2) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data from the bike object.
-
-
-
-
+class Bike
+    def initialize (model, wheels, frame)
+        @model = model
+        @wheels = 2
+        @frame = frame
+        @speed = 0
+    end
+
+    def get_info
+        "The #{@model} bike has #{@wheels} wheels and a #{@frame}cm frame"
+    end
 
 # Expected output example: 'The Trek bike has 2 wheels and a 168cm frame.'
-
+my_bike = Bike.new("Trek", 2, 168)
+puts my_bike.get_info
 
```

Comment:
Since wheels is initialized with a default value of 2, maybe think about an approach that doesn't require passing in the number of wheels each time. Just a thought.